install.js

```diff
@@ -33,42 +33,44 @@ let chromedriver_version = process.env.npm_config_chromedriver_version || proces
 let chromedriverBinaryFilePath;
 let downloadedFile = '';
 
-(async function install() {
-  try {
-    if (detect_chromedriver_version === 'true') {
-      // Refer http://chromedriver.chromium.org/downloads/version-selection
-      const chromeVersion = await getChromeVersion();
-      console.log("Your Chrome version is " + chromeVersion);
-      const chromeVersionWithoutPatch = /^(.*?)\.\d+$/.exec(chromeVersion)[1];
-      await getChromeDriverVersion(getRequestOptions(cdnUrl + '/LATEST_RELEASE_' + chromeVersionWithoutPatch));
-      console.log("Compatible ChromeDriver version is " + chromedriver_version);
-    }
-    if (chromedriver_version === 'LATEST') {
-      await getChromeDriverVersion(getRequestOptions(`${cdnUrl}/LATEST_RELEASE`));
-    } else {
-      const latestReleaseForVersionMatch = chromedriver_version.match(/LATEST_(\d+)/);
-      if (latestReleaseForVersionMatch) {
-        const majorVersion = latestReleaseForVersionMatch[1];
-        await getChromeDriverVersion(getRequestOptions(`${cdnUrl}/LATEST_RELEASE_${majorVersion}`));
+if (require.main === module) {
+  (async function install() {
+    try {
+      if (detect_chromedriver_version === 'true') {
```

Comment:
I only indented the install function execution with this `if`, but the diff doesn't do a good job of showing that